marketplace/accounts/tests/test_views.py

```diff
@@ -0,0 +1,56 @@
+import uuid
+from django.urls import reverse
+
+from marketplace.accounts.views import UserPermissionViewSet, UserViewSet
+from marketplace.core.tests.base import APIBaseTestCase
+from marketplace.accounts.models import User
+
+
+class BaseTest(APIBaseTestCase):
```

Comment:
I don't understand why this class exists, can you explain better. please?

Reply:
I think i would put some code, in this case, this class can be removed.

Reply:
done!